### PR TITLE
Update email setup instructions for Airflow

### PR DESCRIPTION
**Pull request by wolfier**
Update how to setup email for SES and sendgrid in Airflow.

Addresses #466

### CODE REVIEW

File: astro/airflow-alerts.md [hunk 1]
```diff
@@ -42,7 +42,9 @@ To get started with SendGrid:
 
 5. Verify your integration in SendGrid to confirm that the key was activated. If you get an error indicating that SendGrid can't find the test email, try rerunning the cURL code in your terminal before retrying the verification.
 
-6. Open the Airflow UI for your Deployment and [create a connection](https://airflow.apache.org/docs/apache-airflow/stable/howto/connection.html#creating-a-connection-with-the-ui) with the following values:
+6. Open the Variables UI for your Deployment and add a Variable called [AIRFLOW__EMAIL__EMAIL_BACKEND](https://airflow.apache.org/docs/apache-airflow/stable/configurations-ref.html#email-backend) with the value `airflow.providers.sendgrid.utils.emailer.send_email`.
```
Comment (paolaperaza) [inline astro/airflow-alerts.md:45]: @wolfier Thank you for adding this! What do you mean by `Variables UI`? Is this an Airflow Variable or an Astro Env Var on cloud.astronomer.io?

**Review by paolaperaza — COMMENTED** [review]
Quick question for you @wolfier and then we can finish this! I can also port this over to Software version of this doc.

File: astro/airflow-alerts.md (hunk 1 above)
Comment (wolfier) [reply astro/airflow-alerts.md:45]: The Variable UI is the view where you can add (environment) variables to the deployment on the platform

File: astro/airflow-alerts.md [hunk 2]
```diff
@@ -74,7 +76,9 @@ This setup requires an AWS account and use of the [AWS Management Console](https
 
 4. Choose an Amazon EC2 region to use, then write down the code of this server for the next step. Refer to [Amazon's list of available regions and servers](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/using-regions-availability-zones.html#concepts-regions) to determine which server best fits your needs.
 
-5. Open the Airflow UI for your Deployment and [create a connection](https://airflow.apache.org/docs/apache-airflow/stable/howto/connection.html#creating-a-connection-with-the-ui) with the following values:
+5. Open the Variables UI for your Deployment and add a Variable called [AIRFLOW__EMAIL__EMAIL_BACKEND](https://airflow.apache.org/docs/apache-airflow/stable/configurations-ref.html#email-backend) with the value `airflow.providers.amazon.aws.utils.emailer.send_email`.
+
+6. Open the Airflow UI for your Deployment and [create a connection](https://airflow.apache.org/docs/apache-airflow/stable/howto/connection.html#creating-a-connection-with-the-ui) with the following values:
```
Comment (paolaperaza) [inline astro/airflow-alerts.md:81]:
```suggestion
6. In the Airflow UI, [create an Airflow connection](https://airflow.apache.org/docs/apache-airflow/stable/howto/connection.html#creating-a-connection-with-the-ui) with the following values:
```

File: astro/airflow-alerts.md [hunk 3]
```diff
@@ -74,7 +76,9 @@ This setup requires an AWS account and use of the [AWS Management Console](https
 
 4. Choose an Amazon EC2 region to use, then write down the code of this server for the next step. Refer to [Amazon's list of available regions and servers](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/using-regions-availability-zones.html#concepts-regions) to determine which server best fits your needs.
 
-5. Open the Airflow UI for your Deployment and [create a connection](https://airflow.apache.org/docs/apache-airflow/stable/howto/connection.html#creating-a-connection-with-the-ui) with the following values:
+5. Open the Variables UI for your Deployment and add a Variable called [AIRFLOW__EMAIL__EMAIL_BACKEND](https://airflow.apache.org/docs/apache-airflow/stable/configurations-ref.html#email-backend) with the value `airflow.providers.amazon.aws.utils.emailer.send_email`.
```
Comment (paolaperaza) [inline astro/airflow-alerts.md:79]:
```suggestion
5. In the Deployment view of the Cloud UI, create an environment variable with the following values:
    - **Key**: `AIRFLOW__EMAIL__EMAIL_BACKEND`
    - **Value**: `airflow.providers.sendgrid.utils.emailer.send_email`
```

**Review by paolaperaza — COMMENTED** [review]
Clarified paths in Cloud vs. Airflow UI!

File: astro/airflow-alerts.md [hunk 4]
```diff
@@ -74,7 +79,11 @@ This setup requires an AWS account and use of the [AWS Management Console](https
...
 
-5. Open the Airflow UI for your Deployment and [create a connection](https://airflow.apache.org/docs/apache-airflow/stable/howto/connection.html#creating-a-connection-with-the-ui) with the following values:
+5. In the Deployment view of the Cloud UI, create an environment variable with the following values:
+    - **Key**: `AIRFLOW__EMAIL__EMAIL_BACKEND`
+    - **Value**: `airflow.providers.sendgrid.utils.emailer.send_email`
+
+6. In the Airflow UI, [create an Airflow connection](https://airflow.apache.org/docs/apache-airflow/stable/howto/connection.html#creating-a-connection-with-the-ui) with the following values:
 
    - **Connection ID**: `smtp_default`
    - **Connection Type:**: `Email`
```
Comment (paolaperaza) [inline astro/airflow-alerts.md:89]:
```suggestion
   - **Connection Type:** `Email`
```

File: astro/airflow-alerts.md [hunk 5]
```diff
@@ -51,7 +56,7 @@ To get started with SendGrid:
     - **Password**: `<your-api-key>`
```
Comment (paolaperaza) [inline astro/airflow-alerts.md:56]:
```suggestion
    - **Password**: `<your-sendgrid-api-key>`
```

File: astro/airflow-alerts.md [hunk 6]
```diff
@@ -42,7 +42,12 @@ To get started with SendGrid:
 
 5. Verify your integration in SendGrid to confirm that the key was activated. If you get an error indicating that SendGrid can't find the test email, try rerunning the cURL code in your terminal before retrying the verification.
 
-6. Open the Airflow UI for your Deployment and [create a connection](https://airflow.apache.org/docs/apache-airflow/stable/howto/connection.html#creating-a-connection-with-the-ui) with the following values:
+6. In the Deployment view of the Cloud UI, create an environment variable with the following values:
+
+    - **Key**: `AIRFLOW__EMAIL__EMAIL_BACKEND`
+    - **Value**: `airflow.providers.sendgrid.utils.emailer.send_email`
```
Comment (paolaperaza) [inline astro/airflow-alerts.md:48]:
```suggestion
    - **Value**: `airflow.providers.sendgrid.utils.emailer.send_email`
    
    For more information on this environment variable, see [Airflow documentation](https://airflow.apache.org/docs/apache-airflow/stable/howto/email-config.html#send-email-using-sendgrid).
```

File: astro/airflow-alerts.md [hunk 7]
```diff
@@ -74,7 +79,11 @@ This setup requires an AWS account and use of the [AWS Management Console](https
 
 4. Choose an Amazon EC2 region to use, then write down the code of this server for the next step. Refer to [Amazon's list of available regions and servers](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/using-regions-availability-zones.html#concepts-regions) to determine which server best fits your needs.
 
-5. Open the Airflow UI for your Deployment and [create a connection](https://airflow.apache.org/docs/apache-airflow/stable/howto/connection.html#creating-a-connection-with-the-ui) with the following values:
+5. In the Deployment view of the Cloud UI, create an environment variable with the following values:
+    - **Key**: `AIRFLOW__EMAIL__EMAIL_BACKEND`
+    - **Value**: `airflow.providers.sendgrid.utils.emailer.send_email`
```
Comment (paolaperaza) [inline astro/airflow-alerts.md:84]:
```suggestion
    - **Value**: `airflow.providers.amazon.aws.utils.emailer.send_email`
```

File: astro/airflow-alerts.md [hunk 8]
```diff
@@ -74,16 +81,20 @@ This setup requires an AWS account and use of the [AWS Management Console](https
 
 4. Choose an Amazon EC2 region to use, then write down the code of this server for the next step. Refer to [Amazon's list of available regions and servers](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/using-regions-availability-zones.html#concepts-regions) to determine which server best fits your needs.
 
-5. Open the Airflow UI for your Deployment and [create a connection](https://airflow.apache.org/docs/apache-airflow/stable/howto/connection.html#creating-a-connection-with-the-ui) with the following values:
+5. In the Deployment view of the Cloud UI, create an environment variable with the following values:
+    - **Key**: `AIRFLOW__EMAIL__EMAIL_BACKEND`
+    - **Value**: `airflow.providers.amazon.aws.utils.emailer.send_email`
```
Comment (paolaperaza) [inline astro/airflow-alerts.md:86]:
```suggestion
    - **Value**: `airflow.providers.amazon.aws.utils.emailer.send_email`
    
    For more information on this environment variable, see [Airflow documentation](https://airflow.apache.org/docs/apache-airflow/stable/howto/email-config.html#send-email-using-aws-ses).
```

**Review by paolaperaza — APPROVED** [review]
This LGTM! Thanks @wolfier.

Note: @ben-astro Submitted a docs GH issue for this same doc, but I'm gonna go ahead and tackle that in a separate PR. https://github.com/astronomer/docs/issues/631

**Pull request merged**